packages/shim-openclaw-engram/openclaw.plugin.json

```
@@ -105,7 +105,11 @@
     "type": "object",
     "additionalProperties": false,
     "properties": {
-      "abstractionAnchorsEnabled": {
+      "episodicContextEnabled": {
```

Biome (2.5.6)

[error] The key episodicContextEnabled was already declared.

(lint/suspicious/noDuplicateObjectKeys)